laurentlaurent/ShutterFeed-a-soen-341-repo
# 74 send comments to database

## Summary

This PR add the ability to read and write comments

## Related Issue



* Resolves #74
* Resolves #85 
* Resolves #86 

## Testing 

1. Type a comment 
2. See it in action
